lib/stdlib/src/proc_lib.erl

```diff
@@ -265,12 +265,12 @@ exit_p(Class, Reason) ->
 	{M,F,A} when is_atom(M), is_atom(F), is_integer(A) ->
 	    MFA = {M,F,make_dummy_args(A, [])},
 	    crash_report(Class, Reason, MFA),
-	    exit(Reason);
+	    erlang:raise(Class, Reason, erlang:get_stacktrace());
```

Comment:
Done. I didn't think it was really necessary considering the module but better safe than sorry.